action/candidate_transfer_ownership.go

```diff
@@ -0,0 +1,220 @@
+package action
+
+import (
+	"bytes"
+	"math/big"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/iotexproject/iotex-address/address"
+	"github.com/iotexproject/iotex-core/pkg/util/byteutil"
+	"github.com/iotexproject/iotex-core/pkg/version"
```

Comment:
move 2 `iotex-core` lines after L15

Reply:
This will be automatically sorted by vscode go tools when saving

Reply:
vscode can be configured to split imports into three parts as follows:
```
"gopls": {
        "formatting.local": "github.com/iotexproject/iotex-core",
    }
```